backend/src/test/java/com/funeat/member/application/MemberServiceTest.java

```diff
@@ -1,224 +1,286 @@
 package com.funeat.member.application;
 
+import static com.funeat.fixture.MemberFixture.멤버_멤버1_생성;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.SoftAssertions.assertSoftly;
 
 import com.funeat.auth.dto.UserInfoDto;
-import com.funeat.common.DataClearExtension;
+import com.funeat.common.ServiceTest;
 import com.funeat.member.domain.Member;
 import com.funeat.member.dto.MemberProfileResponse;
 import com.funeat.member.dto.MemberRequest;
-import com.funeat.member.persistence.MemberRepository;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.DisplayNameGeneration;
-import org.junit.jupiter.api.DisplayNameGenerator.ReplaceUnderscores;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.transaction.annotation.Transactional;
-
-@Transactional
-@SpringBootTest
-@ExtendWith(DataClearExtension.class)
-@SuppressWarnings("NonAsciiCharacters")
-@DisplayNameGeneration(ReplaceUnderscores.class)
-class MemberServiceTest {
-
-    @Autowired
-    private TestMemberService memberService;
 
-    @Autowired
-    private MemberRepository memberRepository;
+@SuppressWarnings("NonAsciiCharacters")
+class MemberServiceTest extends ServiceTest {
 
     @Nested
     class findOrCreateMember_테스트 {
 
-        @Test
-        void 이미_가입된_사용자면_가입하지_않고_반환한다() {
-            // given
-            final var member = new Member("test", "www.test.com", "1");
-            memberRepository.save(member);
+        @Nested
+        class 성공_테스트 {
 
-            final var userInfoDto = new UserInfoDto(1L, "test", "www.test.com");
-            final var expected = memberRepository.findAll();
+            @Test
+            void 이미_가입된_사용자면_가입하지_않고_반환한다() {
+                // given
+                final var member = 멤버_멤버1_생성();
+                단일_멤버_저장(member);
 
-            // when
-            final var actual = memberService.findOrCreateMember(userInfoDto);
+                final var userInfoDto = new UserInfoDto(1L, "test", "www.test.com");
 
-            // then
-            assertSoftly(softAssertions -> {
-                Assertions.assertFalse(actual.isSignUp());
-                assertThat(expected).containsExactly(actual.getMember());
-            });
-        }
+                final var expected = memberRepository.findAll();
+
+                // when
+                final var actual = memberService.findOrCreateMember(userInfoDto);
+
+                // then
+                assertSoftly(softAssertions -> {
+                    softAssertions.assertThat(actual.isSignUp()).isFalse();
+                    softAssertions.assertThat(expected).containsExactly(actual.getMember());
```

Comment:
```suggestion
                    softAssertions.assertThat(actual.isSignUp())
                        .isFalse();
                    softAssertions.assertThat(expected)
                        .containsExactly(actual.getMember());
```

전체적으로 `assertThat().~`도 점 하나 단위로 개행을 해서 가독성 높이는 건 어떻게 생각하시나요??

Reply:
잊고 있었던 객체지향 생활 체조 원칙 👍